package.json

```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
     "aws-sdk": "2.28.0",
-    "arsenal": "scality/Arsenal#rel/7.1",
+    "arsenal": "scality/Arsenal#03f82ea89145e922bbe08621f19ac596a02235c6",
```

Comment:
why?

Reply:
npm was not installing the latest changes in `rel/7.1` so I added the commit hash of  the latest merge commit for testing.